### ISSUE TITLE
Touhou: Dreaming Butterfly | 东方蝶梦志(941410)

### ISSUE THREAD

**Issue by Seng-Jik** [ValveSoftware/Proton#4481]
# Whitelist Request
- Name of the game to be whitelisted:Touhou: Dreaming Butterfly | 东方蝶梦志
- Steam AppID of the game:941410

## System Information
Gist:https://gist.github.com/Seng-Jik/5c4f1558c2611922a53b8eabb2d1b9a5

## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [x] that no workarounds other than PROTON_* env variables are necessary.

## Issues
- [x] I haven't experienced any issues.
- [x] There are no issues left open for this game.
- [x] Although I consider the gaming experience equal to Windows there are remaining issues:

**Comment by kisak-valve** [comment]
Hello @Seng-Jik, you indicated that this game needs some runtime config options. Can you share what you needed to do in order to get the game to run?

**Comment by Seng-Jik** [comment]
> Hello @Seng-Jik, you indicated that this game needs some runtime config options. Can you share what you needed to do in order to get the game to run?

No, it is from carelessness, it can run in default runtime config, and need no env variable.
